lib/huginn_agent/templates/newagent/spec/spec_helper.rb.tt

```diff
@@ -0,0 +1,7 @@
+base_path = File.expand_path('../../', __FILE__)
```

Comment:
@dsander you're right this is not necessary - I will remove. This is only necessary in the huginn_agent gem I am currently working on, where I'd like to include [`my_agent/spec/support/**/*`](http://stackoverflow.com/a/20615194/809290):
```ruby
Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
```

but `Rails.root` is `spec/huginn', so I've included it as:

```ruby
Dir["#{base_path}/spec/support/**/*.rb"].each { |f| require f }
```